extern/ttconv/pprdrv_tt.cpp

```diff
@@ -1417,22 +1417,6 @@ class StringStreamWriter : public TTStreamWriter
     }
 };
 
-void get_pdf_charprocs(const char *filename, std::vector<int>& glyph_ids, TTDictionaryCallback& dict)
-{
-    struct TTFONT font;
-
-    read_font(filename, PDF_TYPE_3, glyph_ids, font);
-
-    for (std::vector<int>::const_iterator i = glyph_ids.begin();
-            i != glyph_ids.end(); ++i)
-    {
-        StringStreamWriter writer;
```

Comment:
The `StringStreamWriter` above this appears unused as well.

Reply:
indeed; I also removed all the pdf_mode-related patches which came in at the same time.